app/controllers/idv/in_person/state_id_controller.rb

```diff
@@ -57,6 +54,11 @@ def update
           end
 
           idv_session.doc_auth_vendor = Idp::Constants::Vendors::USPS
+
+          analytics.idv_in_person_proofing_state_id_submitted(
+            **analytics_arguments.merge(**form_result),
+          )
```

Comment:
We need to log this _after_ setting `doc_auth_vendor` now.

Reply:
This may or may not be my fault. (Unfortunately, it was the least janky way of doing it.)